docs/source/autoscaling.rst

```diff
@@ -0,0 +1,34 @@
+====================================
+Autoscaling Marathon tasks in PaaSTA
+====================================
+
+PaaSTA allows programmatic control of the number of marathon tasks a service has. It does this by using Zookeeper as a data store to record the number of tasks it thinks should be running for each instance of a service on a per-cluster basis.
+
+How to use autoscaling
+======================
+
+Enabling autoscaling
+--------------------
+
+In order to use autoscaling, edit your ``marathon-*.yaml`` files in your soa configs and add a ``max_instances`` attribute to each instance you want to autoscale. When using autoscaling, the ``instances`` and ``max_instances`` attributes become the minimum and maximum (inclusive) number of marathon tasks paasta will create for your job.
```

Comment:
I still vote for making people do a `min_instances` and `max_instances` if they want autoscaling for real.